### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by bmesuere**
This pull request updates the readme file which contained outdated info. I also restructured the file a bit.

### CODE REVIEW

**Review by niknetniko — APPROVED** [review]
I don't have anything to add, look good to me 👍 

One "gimmick" we could add is a badge about the build status next to the title (as is often done), but it will always be green (I assume) so not that useful.

File: README.md [hunk 1]
```diff
@@ -29,31 +35,36 @@ The documentation of this project can be found at https://docs.dodona.be.
...
-The foreman gem is [not bundled with Dodona](https://github.com/ddollar/foreman/wiki/Don%27t-Bundle-Foreman). Install it globally with `gem install foreman`.
+To start the rails server, delayed job and the webpack dev server at the same time, simply run `bin/server`.
 
-This has one letdown: debugging with `byebug` is broken. You can run `bin/server norails` to only start webpack and delayed_job in foreman and then run `rails s` in a different terminal to be able to use `byebug` again.
+This has one letdown: debugging with `byebug` is broken.
 
-## Localhost subdomain
+#### Localhost subdomain
 
 Dodona use subdomains in order to sandbox exercise descriptions (which are arbitrary HTML pages and could be used for malicious purposes if not properly sandboxed. We serve the main application in development from http://dodona.localhost:3000 and exercise descriptions from http://sandbox.localhost:3000.
```
Comment (beardhatcode) [inline README.md:69]:
```suggestion
Dodona uses subdomains in order to sandbox exercise descriptions (which are arbitrary HTML pages and could be used for malicious purposes if not properly sandboxed. We serve the main application in development from http://dodona.localhost:3000 and exercise descriptions from http://sandbox.localhost:3000.
```

**Comment by beardhatcode** [comment]
Maybe the social image (https://dodona.ugent.be/social-image.png) could be added to the top of the page, or just. It feels a bit off though, due to the different languages and fontsizes.

image

**Comment by chvp** [comment]
I agree that it might be nice to add test status or coverage or something like that. One thing we might also want to do is add how people can reach us to discuss possible contributions or for simple support. Maybe add the dodona@ugent.be email address and mention that we are open to issues and discussions here on GitHub? We could also link to our chat. (The matrix bridge is probably the easiest way to do that, since Slack still doesn't support any open signups.)

**Comment by pdawyndt** [comment]
As an open invitation for new contributers, we might introduce “[good first issue](https://github.blog/2020-01-22-how-we-built-good-first-issues/)” labels. Perhaps rename student label as such.

**Comment by bmesuere** [comment]
> I agree that it might be nice to add test status or coverage or something like that. One thing we might also want to do is add how people can reach us to discuss possible contributions or for simple support. Maybe add the [dodona@ugent.be](mailto:dodona@ugent.be) email address and mention that we are open to issues and discussions here on GitHub? We could also link to our chat. (The matrix bridge is probably the easiest way to do that, since Slack still doesn't support any open signups.)

Feel free to add the matrix info to the contact section. I have addressed the other issues.

**Review by chvp — APPROVED** [review]
I added the matrix info (and two badges).

**Pull request merged**